sonarapp/Program.cs

```diff
@@ -20,6 +20,7 @@ static void Main(string[] args)
 
             Boolean verifiyResult = v.verifywhat(string.Empty);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'verifiyResult' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)